app/Main.hs

```diff
@@ -86,7 +87,7 @@ main = do setLocaleEncoding utf8
                       else return context
           finalContext <- loadFiles context' argFilesToLoad
           case execMode of
-            Repl -> do putStrLn "Welcome to Carp 0.3.0"
+            Repl -> do putStrLn "Welcome to Carp 0.3.1WIP"
```

Comment:
I don’t think that the version string should be changed here. That should happen in master or on release.

Reply:
I think I have an issue with my git repo, or that i'm not using it write. This change was even before my first PR (a commit named WIP i squashed). I have no idea how it got there, maybe during a rebase. It wasn't intended. Version reverted.